command/git_push_test.go

```diff
@@ -78,7 +78,6 @@ func TestGitPush(t *testing.T) {
 			commands := []string{
 				"git checkout master",
 				"git rev-parse master@{upstream}",
-				`git -c "user.name=octocat" -c "user.email=octocat@github.com" commit --file - --author="evergreen <evergreen@mongodb.com>"`,
```

Comment:
I'm not sure if this is actually the right fix or an unrelated hack, but (new) line 84 has 3 commands.